docs/user/agent-deployment.md

```
@@ -326,30 +326,31 @@ aicr diff --baseline baseline.yaml --target current.yaml --fail-on-drift \
 
 ### Job Fails to Start
 
-Check RBAC permissions:
+Check RBAC permissions. The ServiceAccount name is run-scoped (`aicr-<run-id>`), so look it up first:
 ```shell
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)